site/content/en/docs/adoption/_index.md

```diff
@@ -29,6 +29,8 @@ aliases:
 
 **Nvidia**: LeaderWorkerSet deployments are the recommended method for deploying Multi-Node models with NIM, see document [here](https://docs.nvidia.com/nim/large-language-models/1.5.0/deploy-helm.html#multi-node-models).
 
+**Red Hat**: LeaderWorkerSet (LWS) is available in Red Hat OpenShift and can be installed from Redhat Operator Hub with the name "Leader Worker Set", see document [here](https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html-single/ai_workloads/index#leader-worker-set-operator)
```

Comment:
Minor tweaks here:
```suggestion
**Red Hat**: LeaderWorkerSet is available in Red Hat OpenShift and can be installed from Red Hat OperatorHub with the name "Leader Worker Set Operator", see document [here](https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html-single/ai_workloads/index#leader-worker-set-operator).
```

Or, I'd consider this rewording:
```suggestion
**Red Hat**: LeaderWorkerSet is available in Red Hat OpenShift by installing the Leader Worker Set Operator from OperatorHub, see document [here](https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html-single/ai_workloads/index#leader-worker-set-operator). 
```

Reply:
Second options look nice to me. Updated based on this. Thank you.